File: mobile/lib/models/market.dart
class Market {
  final String id;
  final String name;
  final String region;

  Market({required this.id, required this.name, required this.region});

  factory Market.fromJson(Map<String, dynamic> json) {
    return Market(
      id: json['_id'] as String,
      name: json['name'] as String,
      region: json['region'] as String,
    );
  }
}

File: backend/src/models/Market.js
const mongoose = require('mongoose');

const marketSchema = new mongoose.Schema(
  {
    name: { type: String, required: true, trim: true },
    region: { type: String, required: true, trim: true },
  },
  { timestamps: true }
);

module.exports = mongoose.model('Market', marketSchema);
